src/core/unit-serialize.c

```diff
@@ -140,6 +140,8 @@ int unit_serialize_state(Unit *u, FILE *f, FDSet *fds, bool switching_root) {
         (void) serialize_dual_timestamp(f, "condition-timestamp", &u->condition_timestamp);
         (void) serialize_dual_timestamp(f, "assert-timestamp", &u->assert_timestamp);
 
+        (void) serialize_ratelimit(f, "start-ratelimit", &u->start_ratelimit);
```

Comment:
hmm, the cherry on top would be if you'd also add this for `u->auto_start_stop_ratelimit`.

And then also for the `trigger_limit` of Path and Socket objects